packages/gestalt/src/Button.test.js

```diff
@@ -23,12 +23,14 @@ describe('<Button />', () => {
   });
   test('Custom white text color on transparent background', () => {
     const instance = create(<Button color="transparentWhiteText" text="Hello World" />).root;
-    expect(instance.find((element) => element.type === 'div').props.className).toContain('white');
+    expect(instance.findAll((element) => element.type === 'div')[1].props.className).toContain(
```

Comment:
I'm not sure what a better solution is, but using the array index here seems a bit brittle

Reply:
I looked into https://reactjs.org/docs/test-renderer.html#testinstancefindbytype and couldn't find a better alternative to detect the Text cmp